index.bs

```diff
@@ -2888,6 +3055,10 @@ a [=moment=] |triggerTime|, and a [=boolean=] |fenced|:
         return null.
     1. If |aggregatableSourceRegTimeConfig| is not "<code>[=aggregatable source registration time configuration/exclude=]</code>", return null.
     1. Set |triggerContextID| to |value|["<code>[=trigger-registration JSON key/trigger_context_id=]</code>"].
+1. Let |attributionScopes| be null.
+1. If |value|["<code>[=trigger-registration JSON key/attribution_scopes=]</code>"] [=map/exists=]:
+    1. Set |attributionScopes| to the result of running [=parse attribution scopes=] with |value| and [=max number of attribution scopes=].
+1. If |attributionScopes| is an error or [=set/is empty|empty=], return null.
```

Comment:
This works regarding null, but it makes the empty list an error, which is fine if we're not pursuing that behavior.

Reply:
ack.